app/helpers/gov_uk_form_helper.rb

```diff
@@ -21,11 +21,24 @@ def govuk_form_group(show_error_if: nil, hint_id: nil, input: nil, &block)
     )
   end
 
-  def govuk_fieldset_header(text, padding_below: nil)
+  # Either passing in the heading text and let this method sort out its formatting,
+  # or define the heading content manually by wrapping this method around a formated header.
+  # So:
+  #   <%= govuk_fieldset_header 'My Heading' %>
+  #
+  # or
+  #
+  #   <%= govuk_fieldset_header do %>
+  #     <h1 class="govuk-fieldset__heading">My Heading</h1>
+  #   <% end %>
+  #
+  # Both result in the same output
+  def govuk_fieldset_header(text = nil, padding_below: nil, &block)
```

Comment:
I added the wrapping functionality so that both heading (h1 and h2) required on the new page, could be contained within the fieldset legend.